### PR TITLE
Feature/fsar 19 navigation desktop

### PR DESCRIPTION
**Pull request by EmilyHazlehurst**

### CODE REVIEW

File: src/components/general/Navigation/navigation-accessibility.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      }
+    });
+  }
+}
+
+function prevAll(element) {
+  var result = [];
+
+  while (element = element.previousElementSibling)
+    result.push(element);
```
Comment (c-ancia) [inline src/components/general/Navigation/navigation-accessibility.js:66]: I know I'm picky but can you wrap it in {} please?

File: src/components/general/Navigation/navigation-accessibility.js [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  return result;
+}
+
+function nextAll(element) {
+  var result = [];
+
+  while (element = element.nextElementSibling)
+    result.push(element);
```
Comment (c-ancia) [inline src/components/general/Navigation/navigation-accessibility.js:74]: Also here please {}

File: src/components/general/Navigation/navigation-accessibility.js [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+  links.forEach((link) => {
+    link.addEventListener('keydown', e => {
+      if ([37, 38, 39, 40].indexOf(e.keyCode) == -1) {
+        return;
+      }
+      switch (e.keyCode) {
+        case 37: // left arrow
+          // Make sure to stop event bubbling
+          e.preventDefault();
+          e.stopPropagation();
```
Comment (c-ancia) [inline src/components/general/Navigation/navigation-accessibility.js:149]: should we move these 2 lines outside of the switch and place them before it? just so we don't repeat

File: src/components/general/Navigation/navigation-accessibility.js [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+    let next = [];
+    link.addEventListener('keydown', e => {
+      if ([37, 38, 39, 40].indexOf(e.keyCode) == -1) {
+        return;
+      }
+      switch (e.keyCode) {
+        case 37: // left arrow
+          // Make sure to stop event bubbling
+          e.preventDefault();
+          e.stopPropagation();
```
Comment (c-ancia) [inline src/components/general/Navigation/navigation-accessibility.js:91]: should we move these 2 lines outside of the switch and place them before it? just so we don't repeat

**Review by c-ancia — CHANGES_REQUESTED** [review]

**Comment by EmilyHazlehurst** [comment]
Thanks @c-ancia I've made all of those changes and have also made the changes discussed in the call with Anna this morning.

**Review by c-ancia — APPROVED** [review]

**Pull request merged**